### DISCUSSION TITLE
To The .NET Foundation Community

### DISCUSSION THREAD

**Discussion by bjablonsky** [dotnet-foundation/Home#67] [General Discussions]
It's been about 3 weeks since the issues were raised here regarding member projects and the .NET Foundation, and the community responded. I was hoping my feelings on the entire situation would change. They haven't, and I don't think they will in the near future. This post is my own opinion and does not represent the views/opinions of the .NET Foundation or my employer.

I'm really disappointed with the actions of some in the community.

Some background: I've been a .NET Foundation member since open membership was available in 2019. I was excited to join to help out wherever I could and cast my vote for the new member-elected board. After the action groups (now called committees) were announced, I eventually joined the Outreach AG since it was something I was passionate about (being a co-organizer of the NYC .NET Developer's UG). That first year I was barely involved. I didn't know what was going on, and didn't really make an effort to become more involved. I was a bit frustrated because I had an expectation that since the .NET Foundation had existed since 2014, things would have been well organized. I was confused on how to contribute, and it seemed like the updates I was hearing out of action groups was taking forever to get things done. Rather than ask what I could do to help out, I stayed silent. 

Then came an honest discussion on the discussion boards on involvement of the members and the board. After reading and agreeing with many of posts, I came to the realization that I wasn't stepping up to help out. Unfortunately, at this point (mid-2020), the new elections were in full swing, and there was silence while the old board transitioned everything to the new board. Once the new board started ramping things up, I ended up getting involved with the outreach committee more. I volunteered to help out with the .NET Virtual User Groups since I myself was a community organizer. I took a break from assisting with the .NET Foundation in late-2020 as I had the opportunity to create and teach a .NET Bootcamp course to 5 students that I had been working on for a long time. After that wrapped up in December 2020, I pitched the idea to the Outreach committee about taking the course I built and continuing to teach it under the .NET Foundation. They wanted me to pitch the idea to the board in January 2021, and after talking to the board, I took over the defunct education committee. I announced the .NET Foundation Education Committee back in May 2021, and quite a few volunteers reached out to help with planning and discussing the initiative. It's been a ton of work, but it's a great initiative. 

So, why all this background? So you can understand where I'm coming from when I'm explaining why I'm disappointed with the actions of some in the community. I've committed a lot of my time and effort to help make the .NET Foundation better. The .NET Foundation is not perfect. There's a lot that needs improving.

In the original discussion post surrounding the member project issues, I posted a comment encouraging everyone to take a deep breath, not to assume the worst from the .NET Foundation, and to hear out both sides. Bill Wagner (an elected DNF Board Member) had responded earlier that the Board was aware of the issue, and they were discussing it. I asked everyone to give the Board time to respond. The .NET Foundation just had an election, and the new board is just ramping up. I also asked everyone to remember that the board is made up of mostly volunteers. People who volunteered their time and effort to be your elected Board members. They're not being paid. The Board also has to vote on a consensus of what to do. A Board member can't unilaterally make a decision for the foundation without discussing and having the board vote on it. At least that's my interpretation of what the bylaws mean. I felt like that was a reasonable request. Well, apparently some did not feel that was reasonable.

In Claire's apology discussion, Claire had apologized for PR issue, talked about the .NET Foundation member projects, and stated that the board will be holding an open discussion soon to discuss the issues. However, not all the issues that were brought up by the community were addressed. So, it's understandable that some were still frustrated with lack of answers. It was a beginning to an open dialog to discuss what went wrong, what needs to get fixed, and how to do things better in the future. However, instead of giving the time to have these discussions, some proposed that within the next 24 hours the Executive Director needs to resign. Really? It was just over 24 hours from the original discussion post, to the executive director apologizing, to members demanding a resignation. That isn't enough time to have an open dialog about everything and to try to resolve the issues. This was such a knee-jerk reaction to everything.

Then, there were discussions (mostly outside of the GitHub discussions) about getting rid of the .NET Foundation all together. This one really hit home. Really? There's more to the .NET Foundation than just supporting member projects. The .NET Foundation currently pays the Meetup membership fees for 380 .NET user groups worldwide (~$60k/year). The .NET Foundation also runs the .NET Virtual User Group allowing any user group to stream their content via YouTube and reach a broader audience. The .NET Foundation is also helping support the C# Community Discord with Azure credits. The .NET Foundation also put together a speaker directory helping connect speakers to user groups and conferences. They also organized and ran a 24-hour virtual conference (Dev Around The Sun) to fundraise for Direct Relief's COVID-19 emergency fund in May 2020. And, they helped support an education initiative (.NET Foundation Academy) to teach .NET to students. I'm sure there's more I'm missing. Should we just throw all this away?

With all the time and effort I have spent contributing to the .NET Foundation, it's hard to not take any of this on a personal level. I've asked myself this question over the course of the past 3 weeks: do I really want to devote my time and effort to a community where some don't have the patience to have an open and productive discussion to solve an issue? The answer is not really. 

There are serious issues that need to be solved with the foundation. I encourage constructive criticism. Almost all of the discussions so far have been constructive (even if they were impassioned). I personally feel that a lot of this boils down to miscommunication between the .NET Foundation and the member projects/members. The board members/leadership team that I know are passionate about .NET and have nothing but the best intentions. Unfortunately, the .NET Foundation is not their full-time job. They are mostly volunteers devoting what little free time they have to help out the community. This will all take time to fix, and I hope the community understands that. 

Unfortunately, after a lot of thought, I've decided to step away from the .NET Foundation for the time being. I'm stepping down from the Education Committee chair in case anyone is interested in stepping up. I'd be happy to help assist with the transition if the Board approves.

**Comment by rprouse** [comment]
@bjablonsky thank you for everything you've done for the community. I understand and sympathize with your feelings. I hope to see you back in the future in some capacity, your help has been appreciated.

**Comment by uabarahona** [comment]
You have expressed the feelings of many of us that do not necessarily like or know how to talk, thanks. Have a rest is fine, I hope after some time the .NET Foundation could gain a clear north, put some limits and then have you joining again :)

**Comment by jglozano** [comment]
@bjablonsky I want to thank you for the time you spent with the Foundation assisting the community. I completely understand your decision and respect it. Leading a committee is not easy, specially ones that require a lot more time than just attending meetings. Your passion for training and helping others is something I really admire.

**Comment by BillWagner** [comment]
@bjablonsky Thank you for all the time you've committed to this over the past months. I completely understand your decision, but this saddens me. The new education committee is a great example for how we want to start new initiatives: I'm impressed by the strength of the ideas you brought to the board, how we worked together to find others to help, and how much you've all accomplished. 

Thank you again for everything you've done.
